extension/src/XLIFFDocument.ts

```diff
@@ -59,6 +59,11 @@ export class Xliff implements XliffDocumentInterface {
         if (formatXml) {
             xml = this.formatXml(xml);
         }
+        // Workaround "> bug" in xmldom, ref https://github.com/jwikman/nab-al-tools/issues/43
```

Comment:
To make very obvious what's a part of the workaround should we extract it to a function or add a closing comment?

Reply:
Better now?